src/symbolize/gimli/parse_running_mmaps_unix.rs

```diff
@@ -85,16 +85,37 @@ impl FromStr for MapsEntry {
     // e.g.: "ffffffffff600000-ffffffffff601000 --xp 00000000 00:00 0                  [vsyscall]"
     // e.g.: "7f5985f46000-7f5985f48000 rw-p 00039000 103:06 76021795                  /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2"
     // e.g.: "35b1a21000-35b1a22000 rw-p 00000000 00:00 0"
+    //
+    // Note that paths may contain spaces, so we can't use `str::split` for parsing (until
+    // Split::remainder is stabalized #77998).
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let mut parts = s
-            .split(' ') // space-separated fields
-            .filter(|s| s.len() > 0); // multiple spaces implies empty strings that need to be skipped.
```

Comment:
Now this missed that check: multiple continuous empty spaces (but there now test for it)

Reply:
Added a test to verify that multiple continuous spaces work